### PR TITLE
Update startbootstrap-freelance package.json

### PR DESCRIPTION
**Pull request by caio1982**
The startupbootstrap-freelancer theme has been recently updated to cope with the new 4.0.0 version of Bootstrap itself.

Pull request for issue: #12441

### CODE REVIEW

**Comment by Billy4195** [comment]
The version 4 of startbootstrap-freelancer is release on npm package, not GitHub repo, the auto-update configure of startbootstrap-freelancer in `package.json` using GitHub repo to download source code, you should change it to npm auto-update config.
Refer: [auto-update](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md)

File: ajax/libs/startbootstrap-freelancer/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "startbootstrap-freelancer",
   "filename": "js/freelancer.min.js",
-  "version": "3.3.7",
+  "version": "4.0.0",
```
Comment (PeterDaveHello) [inline ajax/libs/startbootstrap-freelancer/package.json]: Thanks for your contribution, however, we can not directly bump the version number, if the auto-update didn't fetch the new versions properly, we should try to dig about the root cause, usually to update the auto-update config. If we really need to add the new version manually, we should also commit the assets.

Comment (sashberd) [reply ajax/libs/startbootstrap-freelancer/package.json]: @PeterDaveHello There are 3 versions 4.0.0 on npm (alpha, beta and release). Should we add all of the manually or only 4.0.0 (release)?
Also may be we can ask author to add git tag of v4.0.0 to git repo and do nothing :)

**Comment by sashberd** [comment]
@caio1982 The v5.0.0 was tagged and hosted by cdnjs.

**Pull request closed without merging**